src/index.ts

```diff
@@ -180,7 +180,7 @@ export function createBirpc<RemoteFunctions = Record<string, never>, LocalFuncti
 
           if (timeout >= 0) {
             timeoutId = setTimeout(() => {
-              reject(new Error(`[birpc] timeout on calling "${method}"`))
+              reject(new Error(`[birpc] timeout on calling "${method}" with arguments "${args.map(stringifyArg)}"`))
```

Comment:
Another solution for this would be to introduce `onTimeoutError` option that would be used here. The current `onError` option does not handle timeout errors.

Then in Vitest we could do something like:

```ts
onTimeoutError(method, args) {
  logger.log(`Test runner timed out while calling ${method} with args ${JSON.stringify(args)}`)
}
```

End users of `birpc` could themselves decide how to handle `args`. 

This would also enable us to identify whether `birpc` timeouts are coming inside worker threads or the main thread - and based on that the error message could be constructed in more detailed way. (cc. @sheremet-va)